flatliners/alertcomparisonscore.py

```diff
@@ -6,13 +6,13 @@
 import flatliners
 
 class AlertComparisonScore(BaseFlatliner):
-    def __init__(self):
+    def __init__(self, max_cache_size: int = 500):
         super().__init__()
 
-        self.score = dict()
-        self.clusters = dict()
-        self.versions = dict()
-        self.alert_deltas = dict()
+        self.score = super().create_cache_dict(maxsize=max_cache_size)
```

Comment:
do we really need `super()` here? I think it's special for the constructor `__init__` call, but other than that it should work with `self.create...`

Reply:
Does it make a difference if the method is a static method?

Reply:
dont know. 
I would make it a member method and not a static one - but up to you